### PR TITLE
active-active fix and other changes

### PR DESCRIPTION
**Pull request by sudhirsinghaqua**
Following changes are included
1. Active-active fix
2. split db
3. external DB

### CODE REVIEW

File: cloudformation/aqua-ecs-ec2/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[![Launch Stack](https://s3.amazonaws.com/cloudformation-examples/cloudformation-launch-stack.png)](https://console.aws.amazon.com/cloudformation/home?#/stacks/new?stackName=aqua-ecs&templateURL=https://s3.amazonaws.com/aqua-security-public/5.3/aquaEcs.yaml)
+[![Launch Stack](https://s3.amazonaws.com/cloudformation-examples/cloudformation-launch-stack.png)](https://console.aws.amazon.com/cloudformation/home?#/stacks/new?stackName=aqua-ecs&templateURL=https://s3.amazonaws.com/aqua-security-public/6.2/aquaEcs.yaml)
```
Comment (yossig-aquasec) [inline cloudformation/aqua-ecs-ec2/README.md]: need to update to 6.0 and not 6.2

File: cloudformation/aqua-ecs-ec2/README.md [hunk 2]
```diff
@@ -102,13 +103,22 @@ When completed, you can obtain the DNS name of the Aqua Server UI from the conso
...
+
+
 # Version upgrade
 
 To upgrade your Aqua Enterprise version, modify the existing stack with the new Aqua product images.
 
 # Enforcer-only deployment
 
-[![Launch Stack](https://s3.amazonaws.com/cloudformation-examples/cloudformation-launch-stack.png)](https://console.aws.amazon.com/cloudformation/home?#/stacks/new?stackName=aqua-ecs&templateURL=https://s3.amazonaws.com/aqua-security-public/5.3/aquaEnforcer.yaml)
+[![Launch Stack](https://s3.amazonaws.com/cloudformation-examples/cloudformation-launch-stack.png)](https://console.aws.amazon.com/cloudformation/home?#/stacks/new?stackName=aqua-ecs&templateURL=https://s3.amazonaws.com/aqua-security-public/6.2/aquaEnforcer.yaml)
```
Comment (yossig-aquasec) [inline cloudformation/aqua-ecs-ec2/README.md]: need to update to 6.0 and not 6.2

File: cloudformation/aqua-ecs-ec2/README.md [hunk 3]
```diff
@@ -166,7 +176,7 @@ It will deploy Aqua Enforcer in your desired cluster and the newly deployed enfo
 
 # Scanner-only Deployment. 
 
-[![Launch Stack](https://s3.amazonaws.com/cloudformation-examples/cloudformation-launch-stack.png)](https://console.aws.amazon.com/cloudformation/home?#/stacks/new?stackName=aqua-ecs&templateURL=https://s3.amazonaws.com/aqua-security-public/5.3/aquaScanner.yaml)
+[![Launch Stack](https://s3.amazonaws.com/cloudformation-examples/cloudformation-launch-stack.png)](https://console.aws.amazon.com/cloudformation/home?#/stacks/new?stackName=aqua-ecs&templateURL=https://s3.amazonaws.com/aqua-security-public/6.2/aquaScanner.yaml)
```
Comment (yossig-aquasec) [inline cloudformation/aqua-ecs-ec2/README.md]: need to update to 6.0 and not 6.2

File: cloudformation/aqua-ecs-ec2/aquaEcs.yaml [hunk 4]
```diff
@@ -600,17 +602,17 @@ Resources:
       TaskRoleArn: !Ref AquaEcsTaskRole
       ExecutionRoleArn: !Ref AquaEcsTaskRole
       RequiresCompatibilities:
-        - EC2
+        - EC2
```
Comment (yossig-aquasec) [inline cloudformation/aqua-ecs-ec2/aquaEcs.yaml:657]: why there is a space in the end of the line?

File: cloudformation/aqua-ecs-ec2/aquaEcs.yaml [hunk 5]
```diff
@@ -676,22 +678,22 @@ Resources:
...
+        MinimumHealthyPercent: 100
       LoadBalancers:
         - ContainerName: !Join 
             - '-'
             - - !Ref ECSClusterName
               - aqua-gateway
-          ContainerPort: '3622'
-          TargetGroupArn: !Ref AquaGatewayTargetGroup
+          ContainerPort: 3622
+          TargetGroupArn: !Ref AquaGatewayTargetGroup
```
Comment (yossig-aquasec) [inline cloudformation/aqua-ecs-ec2/aquaEcs.yaml:748]: why there is a spaces in the end of the line?

File: cloudformation/aqua-ecs-fargate/README.md [hunk 6]
```diff
@@ -1,4 +1,4 @@
-[![Launch Stack](https://s3.amazonaws.com/cloudformation-examples/cloudformation-launch-stack.png)](https://console.aws.amazon.com/cloudformation/home?#/stacks/new?stackName=aqua-ecs&templateURL=https://s3.amazonaws.com/aqua-security-public/aquaFargate.yaml)
+[![Launch Stack](https://s3.amazonaws.com/cloudformation-examples/cloudformation-launch-stack.png)](https://console.aws.amazon.com/cloudformation/home?#/stacks/new?stackName=aqua-ecs&templateURL=https://s3.amazonaws.com/aqua-security-public/6.2/aquaFargate.yaml)
```
Comment (yossig-aquasec) [inline cloudformation/aqua-ecs-fargate/README.md]: please update the value to 6.0

File: cloudformation/aqua-lambda/README.md [hunk 7]
```diff
@@ -1,4 +1,4 @@
-[![Launch Stack](https://s3.amazonaws.com/cloudformation-examples/cloudformation-launch-stack.png)](https://console.aws.amazon.com/cloudformation/home?#/stacks/new?stackName=aqua-serverless&templateURL=https://s3.amazonaws.com/aqua-security-public/aquaServerless.yaml)
+[![Launch Stack](https://s3.amazonaws.com/cloudformation-examples/cloudformation-launch-stack.png)](https://console.aws.amazon.com/cloudformation/home?#/stacks/new?stackName=aqua-serverless&templateURL=https://s3.amazonaws.com/aqua-security-public/6.2/aquaServerless.yaml)
```
Comment (yossig-aquasec) [inline cloudformation/aqua-lambda/README.md:1]: please update the value to 6.0

File: cloudformation/aqua-ecs-ec2/README.md [hunk 8]
```diff
@@ -23,6 +23,7 @@ A CloudFormation template is used to deploy Aqua Enterprise. This can be done ei
 
  - An ECS cluster with at least 2 instances registered
  - A VPC with at least 2 subnets
+ - A certificate via AWS Certificate Manager
```
Comment (yossig-aquasec) [inline cloudformation/aqua-ecs-ec2/README.md:26]: if we giving a yml with a none ssl, I think that we need to update the guide to something like this:
A certificate via AWS Certificate Manager or use the yml without the ssl. wdyt?

**Review by yossig-aquasec — APPROVED** [review]
approved

**Review by yossig-aquasec — DISMISSED** [review]
approved

**Review by rshmiel — APPROVED** [review]

**Pull request merged**